charts/testkube-operator/templates/deployment.yaml

```diff
@@ -70,26 +72,26 @@ spec:
         livenessProbe:
           httpGet:
             path: /healthz
-            port: {{ .Values.livenessProbePort }}
-          initialDelaySeconds: {{ .Values.initialDelaySeconds }}
-          periodSeconds: {{ .Values.periodSeconds }}
+            port: 8081
+          initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+          periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
         readinessProbe:
           httpGet:
             path: /readyz
-            port: {{ .Values.readinessProbePort }}
-          initialDelaySeconds: {{ .Values.readinessProbeinitialDelaySeconds }}
-          periodSeconds: {{ .Values.readinessProbeperiodSeconds }}
+            port: 8081
```

Comment:
why is it hardcoded? users asked about ability to change this

Reply:
It wasn't configured before to be changeable, i.e. we could change this value but we didnt change the `--health-probe-bind-address=:8081` flag.

If it needs to be configurable we need to change the flag also to support it.

Reply:
Ok, I've made it configurable